connect/src/routes/tokenBridge/automatic.ts

```diff
@@ -89,7 +89,12 @@ export class AutomaticTokenBridgeRoute<N extends Network>
     fromChain: ChainContext<N>,
     toChain: ChainContext<N>,
   ): Promise<TokenId[]> {
-    return [await TokenTransfer.lookupDestinationToken(fromChain, toChain, sourceToken)];
+    try {
+      return [await TokenTransfer.lookupDestinationToken(fromChain, toChain, sourceToken)];
+    } catch (e) {
+      console.error(`Failed to get destination token: ${e}`);
```

Comment:
I dont love the console.errors, but we need a general strat around what/when/where we console.error

Reply:
better would be to not even throw here and just return something akin to a `Result` (`Some(token) | None`)